services/pane.js

```diff
@@ -148,6 +151,28 @@ function openPublish() {
   });
 }
 
+/**
+ * open new page type dialog pane
+ * @returns {Promise}
+ */
+function openNewPage() {
+  var header = 'Select New Page Type',
```

Comment:
Messed around with this some more. Prefer "New Page" too, as it's fast. But those words as a design element feel a little off.